src/Playroom/RenderCode/RenderCode.tsx

```diff
@@ -9,7 +9,13 @@ import {
   ReactFragmentPragma,
 } from '../../utils/compileJsx';
 
-export default function RenderCode({ code, scope }) {
+export default function RenderCode({
+  code,
+  scope,
+}: {
+  code: string | undefined;
+  scope: Record<string, ComponentType>;
```

Comment:
I don't think `ComponentType` is correct here. You can inject whatever you want with `scope`, a constant, function, component, etc. Maybe `Record<string, any>` would be more appropriate.